File: app/src/main/java/com/example/jianming/myapplication/Local1KActivity.java
package com.example.jianming.myapplication;

import android.content.Context;
import android.content.Intent;
import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;


public class Local1KActivity extends AppCompatActivity {

    Context self = this;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_local_1k);
        findViewById(R.id.local1000).setOnClickListener(v -> goToLocal1000());


    }

    public void goToLocal1000() {
        Intent intent = new Intent(self, PicAlbumListActivity.class);
        startActivity(intent);
    }

}

File: app/src/main/java/com/example/jianming/myapplication/Local1KActivity.kt
package com.example.jianming.myapplication

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.jianming.myapplication.databinding.ActivityLocal1kBinding

class Local1KActivity : AppCompatActivity() {

    private lateinit var binding: ActivityLocal1kBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivityLocal1kBinding.inflate(layoutInflater)
        val view = binding.root
        setContentView(view)
        binding.local1000.setOnClickListener {
            startActivity(Intent().setClass(this, PicSectionListActivity::class.java))
        }

        binding.offline.setOnClickListener {
            val intent = Intent()
                .setClass(this, PicSectionListActivity::class.java)
                .putExtra("offline", true)
            startActivity(intent)
        }

    }
}